src/Sylius/Component/Core/Cart/Provider/NewCartProvider.php

```diff
@@ -61,7 +63,11 @@ public function getCart()
             throw new \LogicException('Decorated cart provider must return a cart instance, null given.');
         }
 
-        $cart->setChannel($this->shopperContext->getChannel());
+        try {
```

Comment:
This should be removed after we figure out how to run ``UserCartRecalculationListener`` and ``SessionCartSubscriber`` only on shop side.